crates/nargo/src/cli/prove_cmd.rs

```diff
@@ -27,6 +28,14 @@ pub(crate) fn run(args: ArgMatches) -> Result<(), CliError> {
 /// So when we add witness values, their index start from 1.
 const WITNESS_OFFSET: u32 = 1;
 
+// A map from the fields in an TOML/JSON file
+// which correspond to some ABI to their values
```

Comment:
```suggestion
/// A map from the fields in an TOML/JSON file
/// which correspond to some ABI to their values
```

This is more general across the codebase but what are your thoughts on turning comments like this into doc comments? We'll then get this showing up on cursor hover.

Reply:
Sounds good to me